pkg/pattern/aggregation/push.go

```diff
@@ -287,6 +297,31 @@ func (p *Push) run(pushPeriod time.Duration) {
 	}
 }
 
+func (p *Push) sendPayload(ctx context.Context, payload []byte) (int, error) {
+
+	status, err := p.send(ctx, payload)
+	if err != nil {
+		errorType := "unknown"
+		if status == 429 {
+			errorType = "rate_limited"
```

Comment:
could we move these to constants so their easier to find, and test against?

Reply:
Yep, moved this to constants